FV3/atmos_cubed_sphere/driver/fvGFS/atmosphere.F90

```diff
@@ -188,6 +188,7 @@ module atmosphere_mod
 use fv_update_phys_mod, only: fv_update_phys
 use fv_nwp_nudge_mod,   only: fv_nwp_nudge_init, fv_nwp_nudge_end, do_adiabatic_init
 use fv_io_mod,          only: fv_io_register_nudge_restart
+use fv_mapz_mod,        only: moist_cv
```

Comment:
Is this supposed to be the fmapz one or the new one in GFS_physics?

Reply:
This actually isn't needed at all here (leftover from earlier stale commits)!  Removing.